src/services/DocumentSelectionRequestProcessor.ts

```diff
@@ -122,10 +122,17 @@ export class DocumentSelectionRequestProcessor {
   			}
   		}
 
+			const sourceIp = event.requestContext.identity?.sourceIp;
```

Comment:
As discussed, this is redundant and not required because the /session endpoint on F2F will get this ip and update it in the session store as clientIpAddress which can be used below to create the TXMA event.